lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +20,17 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str):
+        return None
+
+    symbol_list = list(text)
```

Comment:
don't creat list, try to use replace string method

Reply:
Done